tests/test_vectorized_mobject.py

```diff
@@ -85,6 +85,20 @@ def test_vgroup_remove_dunder():
     assert len(obj.submobjects) == 0
 
 
+def test_vmob_add_to_back():
```

Comment:
This test is perfect but why not testing the TypeError thrown [there](https://github.com/ManimCommunity/manim/pull/1194/files#diff-fa8fe466fb7275e64543789145cbd2e35b267715610a6376591ddbda0d2f6194R367) as well ?